tests/test_oic_message.py

```diff
@@ -953,3 +962,112 @@ def test_verify_token_encrypted_no_key():
         verify_id_token(msg, keyjar=KeyJar(),
                         iss="https://sso.qa.7pass.ctf.prosiebensat1.com",
                         client_id="554295ce3770612820620000")
+
+
+def test_logout_token():
```

Comment:
Please make a class for each message and add individual scenarios as test cases. That makes it much clearer which part is failing.

Reply:
OK